ci/pipeline.yml

```diff
@@ -0,0 +1,119 @@
+---
+resource_types:
+  - name: gcs-resource
+    type: docker-image
+    source:
+      repository: frodenas/gcs-resource
+
+resources:
+  - name: gcp-service-broker
+    type: git
+    source:
+      uri: https://github.com/GoogleCloudPlatform/gcp-service-broker.git
+      branch: concourse
```

Comment:
Is the `concourse` branch like a `develop` branch? We use `develop` and `master` in other projects.